<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-16T12:37:02 IST" name="Suite" finished-at="2025-02-16T12:37:02 IST" duration-ms="138">
    <groups>
    </groups>
    <test started-at="2025-02-16T12:37:02 IST" name="demo" finished-at="2025-02-16T12:37:02 IST" duration-ms="50">
      <class name="Test.test3">
        <test-method is-config="true" signature="first()[pri:0, instance:Test.test3@7c729a55]" started-at="2025-02-16T12:37:02 IST" name="first" finished-at="2025-02-16T12:37:02 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- first -->
        <test-method signature="mid()[pri:0, instance:Test.test3@7c729a55]" started-at="2025-02-16T12:37:02 IST" name="mid" finished-at="2025-02-16T12:37:02 IST" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mid -->
        <test-method is-config="true" signature="last()[pri:0, instance:Test.test3@7c729a55]" started-at="2025-02-16T12:37:02 IST" name="last" finished-at="2025-02-16T12:37:02 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- last -->
      </class> <!-- Test.test3 -->
    </test> <!-- demo -->
    <test started-at="2025-02-16T12:37:02 IST" name="Test" finished-at="2025-02-16T12:37:02 IST" duration-ms="15">
      <class name="Test.test2">
        <test-method is-config="true" signature="test3()[pri:0, instance:Test.test2@4efbca5a]" started-at="2025-02-16T12:37:02 IST" name="test3" finished-at="2025-02-16T12:37:02 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method signature="nrml()[pri:0, instance:Test.test2@4efbca5a]" started-at="2025-02-16T12:37:02 IST" name="nrml" finished-at="2025-02-16T12:37:02 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- nrml -->
      </class> <!-- Test.test2 -->
      <class name="Test.Demo">
        <test-method is-config="true" signature="no1()[pri:0, instance:Test.Demo@6069db50]" started-at="2025-02-16T12:37:02 IST" name="no1" finished-at="2025-02-16T12:37:02 IST" duration-ms="7" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- no1 -->
        <test-method signature="hello()[pri:0, instance:Test.Demo@6069db50]" started-at="2025-02-16T12:37:02 IST" name="hello" finished-at="2025-02-16T12:37:02 IST" duration-ms="4" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Test.Demo.hello(Demo.java:12)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hello -->
        <test-method is-config="true" signature="last1()[pri:0, instance:Test.Demo@6069db50]" started-at="2025-02-16T12:37:02 IST" name="last1" finished-at="2025-02-16T12:37:02 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- last1 -->
      </class> <!-- Test.Demo -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
